src/igraph/drawing/__init__.py

```diff
@@ -423,6 +423,10 @@ def plot(obj, target=None, bbox=(0, 0, 600, 600), *args, **kwds):
     @param target: the target where the object should be plotted. It can be one
       of the following types:
 
+        - C{matplotib.axes.Axes} -- a matplotlib/pyplot axes in which the
+          graph will be plotted. Drawing, saving to file, etc. is mostly
```

Comment:
As a novice reading this, my immediate question is: what does "mostly" mean here?